op-challenger/fault/monitor.go

```diff
@@ -50,7 +52,8 @@ func (m *gameMonitor) progressGames(ctx context.Context) error {
 	if err != nil {
 		return fmt.Errorf("failed to load current block number: %w", err)
 	}
-	games, err := m.source.FetchAllGamesAtBlock(ctx, new(big.Int).SetUint64(blockNum))
+	earliest := time.Now().Unix() - int64(m.gameWindow.Seconds())
```

Comment:
Personally, I'd have written this as:
```suggestion
	earliest := time.Now().Sub(m.gameWindow).Unix()
```

But feel free to ignore that.

Reply:
Actually, this definitely should use a `Clock` rather than calling `time.Now()` directly. Then you can control time in unit tests as well.